modules/chapter3/pages/section3.adoc

```diff
@@ -0,0 +1,76 @@
+These images are designed for data scientists, providing them with the necessary tools to start working quickly.
+Moreover, Red{nbsp}Hat provides support for these images for at least one year, and releases major versions of each image typically every six months.
+
+By default, RHODS includes these images:
+
+[compact]
+* CUDA
+* Standard Data Science
+* TensorFlow
+* PyTorch
+* Minimal Python
+* TrustyAI
+
+// Explain what exactly is a notebook image that comes out of the box with RHODS. Provide links to the Containerfile/Dockerfile for the images shipped with RHODS
+// References to the list of base container images provided by RHODS - product docs links
+
+These images are regular container images.
+The source Dockerfiles for these images are available at the https://github.com/red-hat-data-services/notebooks/tree/main/jupyter[red-hat-data-services/notebooks] GitHub repository.
+
+These images are also available in the Quay.io registry at https://quay.io/organization/modh.
+RHODS tracks these images by using ImageStreams that point to the notebook images available in Quay.
+You can inspect the available images by running the following command:
+
+[subs=+quotes]
+----
+$ *oc get imagestream -n redhat-ods-applications*
+NAME                                IMAGE REPOSITORY                                                                                                                                 TAGS                                                               UPDATED
+pytorch                             default-route-openshift-image-registry.apps.rhods-internal.61tk.p1.openshiftapps.com/redhat-ods-applications/pytorch                             2023.1,1.2-cuda-11.4,2023.1-cuda-11.7 + 3 more...                  6 weeks ago
+s2i-generic-data-science-notebook   default-route-openshift-image-registry.apps.rhods-internal.61tk.p1.openshiftapps.com/redhat-ods-applications/s2i-generic-data-science-notebook   2023.1,1.2,py3.9-v2,py3.8-v1                                       6 weeks ago
+s2i-minimal-notebook                default-route-openshift-image-registry.apps.rhods-internal.61tk.p1.openshiftapps.com/redhat-ods-applications/s2i-minimal-notebook                2023.1,1.2,py3.9-v2,py3.8-v1                                       6 weeks ago
+tensorflow                          default-route-openshift-image-registry.apps.rhods-internal.61tk.p1.openshiftapps.com/redhat-ods-applications/tensorflow                          2023.1,1.2,2023.1-cuda-11.8,1.2-cuda-11.4 + 2 more...              6 weeks ago
+...
+----
+
+For more information about notebook images, refer to the https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html-single/working_on_data_science_projects/index#notebook-images-for-data-scientists_notebooks[_Notebook images for data scientists_ section in the Red{nbsp}Hat OpenShift Data Science documentation].
+
+== Customizing Workbenches with JupyterLab
```

Comment:
Can you add a prefix like "Exercise:" or "Lab:" before the title to indicate to students that they have to consider this as a hands-on lab that they need to do.

Reply:
Sure!